### DISCUSSION TITLE
Welcome to KahootGPT Discussions!

### DISCUSSION THREAD

**Discussion by itsmarsss** [itsmarsss/kahootgpt#12] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PetitPotiron** [comment]
Hi! I must say that this is maybe the most surprising ChatGPT browser extension! I would never have thought of that! Unfortunately, I don't play Kahoots anymore...
